org.kde.kate.json

```diff
@@ -0,0 +1,46 @@
+{
+    "id": "org.kde.kate",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.13",
+    "sdk": "org.kde.Sdk",
+    "command": "kate",
+    "rename-icon": "kate",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=cups",
```

Comment:
does the print portal not work for this use case?

Reply:
The print portal doesn't work on Plasma. I don't recall the details but i think that the gist of it is that the API it provides is not good for us, so if you're on gnome it's fine because the print portal api was though for the glib printing API but for us it's kind of hard/impossible to implement.

@grulja knows more

Reply:
Reason for this is that Qt doesn't support print portal at this moment so if you want to print a file, using this socket is your only option. The portal API is fine, problem is Qt implementation of printing, which is split into plugins and to make it to use the portals internally would require significat code changes in my opinion so that's why nobody implemented it so far.